scripts/compile_cost_assumptions.py

```diff
@@ -216,9 +216,44 @@ def get_data_DEA(tech, data_in):
     return df_final
 
 #
+
+def add_offwind_connection_data(costs):
+    """"
+    add connection costs for offshore wind from DEA
+    """
+    # station includes platform cost, project management and environmental
+    # assessment and transformer station cost
+    costs.loc[('offwind-ac-station', 'investment'), 'value'] = 260
```

Comment:
maybe add as a comment how this value is calculated (costs offshore platform + project managment and environmental assessment + transformer station onshore)